### PR TITLE
added 2.1 preview SDK for text analytics

### PR DESCRIPTION
**Pull request by laramume**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
[Rest API PR](https://github.com/Azure/azure-rest-api-specs/pull/4062)
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by dsgouda — CHANGES_REQUESTED** [review]
@laramume Could you tell us why you have created a new directory named `-2.1-preview`

**Comment by laramume** [comment]
@dsgouda We have two versions right now(stable and preview) which you can take a look at [here](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/cognitiveservices/data-plane/TextAnalytics). We needed two separate SDK's for it that's why I created a separate directory and generated new code in a new project.

**Comment by dsgouda** [comment]
We do not allow multiple versions of SDK in the repo. We need to understand your requirements offline here

**Comment by laramume** [comment]
@dsgouda based on the discussion I have made it to one sdk instead of two.

File: src/SDKs/_metadata/cognitiveservices_data-plane_TextAnalytics.txt [hunk 1]
```diff
@@ -3,18 +3,12 @@ AutoRest installed successfully.
 Commencing code generation
 Generating CSharp code
 Executing AutoRest command
-cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TextAnalytics/readme.md --csharp --version=latest --reflect-api-versions --csharp.output-folder=E:\azure-sdk-for-net\src\SDKs\CognitiveServices\dataPlane\Language\TextAnalytics\TextAnalytics\Generated\TextAnalytics
-2018-07-26 19:20:31 UTC
-1) azure-rest-api-specs repository information
+cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TextAnalytics/readme.md --csharp --version=latest --reflect-api-versions --tag=release_2_1 --csharp.output-folder=C:\azure-sdk-for-net\src\SDKs\CognitiveServices\dataPlane\Language\TextAnalytics\TextAnalytics-2.1-preview\Generated\TextAnalytics-2.1-preview
```
Comment (dsgouda) [inline src/SDKs/_metadata/cognitiveservices_data-plane_TextAnalytics.txt]: This log looks incorrect. please regenerate the code. The `csharp.output-folder` should not be `TextAnalytics-2.1-preview`

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please link the related REST spec PR

**Comment by laramume** [comment]
@dsgouda Added the rest api PR and also updated the log files by regenerating the code

**Comment by dsgouda** [comment]
@laramume please update the `Version` and `PackageReleaseNotes` as we discussed offline

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**